x-pack/plugins/siem/public/components/open_timeline/open_timeline.tsx

```diff
@@ -98,9 +97,9 @@ export const OpenTimeline = React.memo<OpenTimelineProps>(
 
     const onRefreshBtnClick = useCallback(() => {
       if (refetch != null) {
-        refetch();
+        refetch(searchResults, totalSearchResultsCount);
```

Comment:
This is to fix the flashing timelines' table while refetching data, not a good fix, so I'm opening an issue to follow it up